test/examples/otl-resolution-errors.json

```diff
@@ -0,0 +1,103 @@
+{
+    "@context": "../../docs/context/v0.3.0/phyx.json",
```

Comment:
Shouldn't this be using the canonical HTTP URI instead? Any reason to make this deviate from what folks out there should be doing, and what we say in the paper folks should be doing?

Reply:
Good catch! I copied this from another JSON file that needs to point to the local Phyx context so that we can test it, but this file doesn't need that. Fixed in f03a1f6.